components/TradingCardNumResults.jsx

```diff
@@ -0,0 +1,7 @@
+export default function TradingCardNumResults({ numResults, numTotal }) {
+  return (
+    <div className="cmp-trading-card-num-results" aria-live="polite" role="status">
```

Comment:
As far as I can tell from light research, this seems correct, and if not, we can fix it later!